public/apps/configuration/panels/audit-logging/audit-logging.tsx

```diff
@@ -56,17 +64,15 @@ export function AuditLogging(props: AppDependencies) {
     fetchData();
   }, [props.coreStart.http]);
 
-  const statusPanel = renderStatusPanel(onSwitchChange, auditLoggingEnabled);
+  const statusPanel = renderStatusPanel(onSwitchChange, configuration.enabled || false);
+
+  if (!configuration.enabled) {
+    return statusPanel;
+  }
 
   return (
     <>
-      <EuiPanel>
-        <EuiTitle>
-          <h3>Audit logging</h3>
-        </EuiTitle>
-        <EuiHorizontalRule />
-        {statusPanel}
-      </EuiPanel>
+      {statusPanel}
```

Comment:
I didn't get it. Moving title to status panel means if `!configuration.enabled`, the title should not show>

Reply:
Demoed offline. The title is part of the status panel per the mockup.

Reply:
Yep, there are 2 returns. At line 69~71 returns statusPanel (contains title) if `!configuration.enabled`, and other wise line 75 will also return statusPanel (contains title).